framework/src/modules/chain/transport/transport.js

```diff
@@ -63,20 +62,19 @@ class Transport {
 
 		this.constants = {
 			broadcasts,
-			maxSharedTransactions,
 		};
 
 		this.transactionPoolModule = transactionPoolModule;
 		this.blocksModule = blocksModule;
 		this.processorModule = processorModule;
 
-		this.broadcaster = new Broadcaster(
-			this.constants.broadcasts,
-			this.transactionPoolModule,
-			this.logger,
-			this.channel,
-			this.storage,
-		);
+		this.broadcaster = new Broadcaster({
+			broadcasts: this.constants.broadcasts,
```

Comment:
is `broadcasts` a flag?

Reply:
broadcasts directly comes from the config, which is `object` 😢